examples/spherical_expansion_example.ipynb

```diff
@@ -0,0 +1,379 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# SOAP example"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To install rascal:\n",
+    "+ mkdir ../build \n",
+    "+ cd build\n",
+    "+ cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=ON ..\n",
+    "+ make -j 4\n",
+    "+ make install"
```

Comment:
The instructions might be out of date; I didn't use them. Try the standard `make install` to install the Python bindings into your preferred Python environment, then run the notebook using the exact same environment -- it should work. And yes, the timings will be updated when we compare against either quippy (unlikely, since it's still Py2k-only) or another Python SOAP implementation.